pmd-kotlin/src/main/antlr4/net/sourceforge/pmd/lang/kotlin/ast/KotlinLexer.g4

```diff
@@ -1,1668 +1,17 @@
 /**
```

Comment:
I think, this file doesn't need to be executable (755)

Reply:
agree